acceptance/bundle/deploy/pipeline/auto-approve/test.toml

```diff
@@ -1,6 +1,8 @@
 Local = true
 Cloud = true
 
+EnvMatrix.DATABRICKS_CLI_DEPLOYMENT = ["terraform"]  # requires "bundle summary"
```

Comment:
Nit: mix of preceding line and suffix comments.

Do you have a list of these missing features or intend to grep for this `EnvMatrix` to get the accurate list?

Reply:
Going to grep. Update comments to be on the same line.